### ISSUE TITLE
Incorrect main/module/exports specified in package.json, possibly

### ISSUE THREAD

**Issue by josephfh** [InsightSoftwareConsortium/xstate-lit#14]
_Using pnpm in a fresh https://astro.build/ repo_

I'm having an issue with importing the library like this
`import { SelectorController } from "xstate-lit";`

This is a workaround
`import { SelectorController } from "../../node_modules/xstate-lit/dist/select-controller";`

```
web-astro-lit:dev: Failed to resolve entry for package "xstate-lit". The package may have incorrect main/module/exports specified in its package.json.
web-astro-lit:dev: 8:36:54 AM [vite] Error when evaluating SSR module /Users/user/code/prototyping/framework-performance-comparison/apps/web-astro-lit/src/pages/index.astro: failed to import "/src/components/test.ts"
```

<img width="1111" alt="image" src="https://github.com/InsightSoftwareConsortium/xstate-lit/assets/889121/90725166-a62f-43af-acb6-7e200d89c536">

**Comment by PaulHax** [comment]
Thanks for the report and workaround.  I guess `files` is only sufficent for some bundlers:
https://github.com/InsightSoftwareConsortium/xstate-lit/blob/main/package.json#L5-L7

Do you know what the `exports` section should be?  https://nodejs.org/api/packages.html#package-entry-points

**Comment by josephfh** [comment]
Adding an index.ts and some extra info in package.json seems to fix it under my pnpm :)

Pull request here: https://github.com/InsightSoftwareConsortium/xstate-lit/pull/15

**Comment by PaulHax** [comment]
Addressed with #15
